browser-test/src/support/admin_questions.ts

```diff
@@ -33,4 +54,52 @@ export class AdminQuestions {
     expect(tableInnerText).toContain(questionName)
     expect(tableInnerText).toContain(questionText)
   }
+
+  async addNumberQuestion(questionName: string,
+    description = 'number description',
+    questionText = 'number question text',
+    helpText = 'number question help text') {
+    await this.page.click('text=Questions')
+    await this.page.click('#create-question-button')
+
+    await this.page.click('#create-number-question')
+
+    await this.page.fill('text="Name"', questionName)
+    await this.page.fill('text=Description', description)
+    await this.page.fill('text=Question Text', questionText)
+    await this.page.fill('text=Question help text', helpText)
+
+    await this.page.click('text=Create')
+
+    expect(await this.page.innerText('h1')).toEqual('All Questions')
+
+    const tableInnerText = await this.page.innerText('table')
+
+    expect(tableInnerText).toContain(questionName)
+    expect(tableInnerText).toContain(questionText)

```

Comment:
suggestion (non-blocking): Could we extract these lines to a helper function, since they appear to be the same for all types of questions?

Reply:
Sure. If you don't mind, I'd like to do it in a followup PR.

Reply:
No worries! I'm actually trying to write a browser test now, and might take back my suggestion - as soon as we have types with custom config (like multi-option questions), the form-filling looks different. Not sure if it's worth it to have a helper function for asserting question exists or filling in the question form basics

Reply:
Yeah, I am also thinking about having a helper just for common fields and each question type can augment as needed. And good idea for an assertion helper as well.